jupyterhub/templates/proxy/deployment.yaml

```diff
@@ -9,12 +9,6 @@ metadata:
   name: proxy
 spec:
   replicas: 1
-  strategy:
```

Comment:
why removing the rolling update on the proxy as well? There is not the same potential issue and it does have a livenessprobe to switch at the right moment

Reply:
I agree, but I think the proxy might *already* have this behavior, and my patch had the opposite effect of what the comment implies. The hub is constrained by the PVC binding, but the proxy isn't - so the defaults should 'just work'.